cmd/coordinator.go

```diff
@@ -217,6 +224,227 @@ var estimateDepositsSweepFeeCommandDescription = "Estimates the satoshi " +
 	"legacy P2SH deposits. If the estimated fee exceeds the maximum fee " +
 	"allowed by the Bridge contract, the maximum fee is returned as result"
 
+var submitDepositSweepProofCommand = cobra.Command{
+	Use:              "submit-deposit-sweep-proof",
+	Short:            "submit deposit sweep proof",
+	Long:             submitDepositSweepProofCommandDescription,
+	TraverseChildren: true,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		ctx := cmd.Context()
+
+		_, tbtcChain, _, _, _, err := ethereum.Connect(
+			ctx,
+			clientConfig.Ethereum,
+		)
+		if err != nil {
+			return fmt.Errorf(
+				"could not connect to Ethereum chain: [%v]",
+				err,
+			)
+		}
+
+		btcChain, err := electrum.Connect(ctx, clientConfig.Bitcoin.Electrum)
+		if err != nil {
+			return fmt.Errorf("could not connect to Electrum chain: %v", err)
+		}
+
+		transactionHashFlag, err := cmd.Flags().GetString(transactionHashFlagName)
+		if err != nil {
+			return fmt.Errorf("failed to find transaction hash flag: %v", err)
+		}
+
+		logger.Infof(
+			"Preparing deposit sweep proof for transaction: %s",
+			transactionHashFlag,
+		)
+
+		transactionHash, err := bitcoin.NewHashFromString(
+			transactionHashFlag,
+			bitcoin.ReversedByteOrder,
+		)
+		if err != nil {
+			return fmt.Errorf("failed to parse transaction hash flag: %v", err)
+		}
+
+		txProofDifficulty, err := tbtcChain.TxProofDifficultyFactor()
+		if err != nil {
+			return fmt.Errorf(
+				"failed to get transaction proof difficulty factor: %v",
+				err,
+			)
+		}
+
+		// Increase the required confirmations by one. The Bridge calculates
+		// the required difficulty of a chain of block headers by multiplying
+		// the difficulty of the first block header by the difficulty factor.
+		// If the block headers happen to span the Bitcoin epoch difficulty
+		// change and there is a drop of difficulty between the epochs, the sum
+		// of difficulties from the headers chain may be too low. Adding one
+		// more block header will ensure the sum of difficulties is high enough.
+		requiredConfirmations := uint(txProofDifficulty.Uint64()) + 1
```

Comment:
This is because of this piece of code https://github.com/keep-network/tbtc-v2/blob/66a04c618e140230d25ea0d784ed75e6ba3030a2/solidity/contracts/bridge/BitcoinTx.sol#L255.